addons/gsri_insignias/config.cpp

```diff
@@ -0,0 +1,99 @@
+class CfgPatches {
+  class gsri_props {
+    author="www.gsri.team";
```

Comment:
Use a single stringtable instead ? Eg. $STR_GSRI_AUTHORS is already in use for GSRI Gear addon. It can be reused without redeclaration here.

Reply:
Are you certain it works although not in the same PBO ?

Reply:
I think so

Reply:
Tested and works.
Arma 3 is so weird ...